src/pages/metapackages/commerce/index.md

```diff
@@ -52,7 +52,19 @@ At this point, you should see symlinks for all the `pwa-commerce` modules inside
 -  Run a Magento installation with additional modules.
 -  Develop locally using the standard git workflow.
 
-You may need to ensure that there are no `Magento_PWA*` modules listed as `enabled` when you run `bin/magento module:status`. If there are, [follow the docs on how to enable modules](https://developer.adobe.com/commerce/php/development/build/component-management/).
+You may need to ensure that there are no `Magento_*Pwa` modules listed as `disabled` when you run `bin/magento module:status`. If there are, [follow the docs on how to enable modules](https://developer.adobe.com/commerce/php/development/build/component-management/).
+
+Note: You should expect to see the following modules
+
+    Magento_ContactGraphQlPwa
+    Magento_NewsletterGraphQlPwa
+    Magento_PageBuilderPwa
+    Magento_ReCaptchaGraphQlPwa
+    Magento_ReCaptchaPwa
+    Magento_UrlRewriteGraphQlPwa
+    *REVIEWER* I only have access to run the open source one, do we want to update this too? 
```

Comment:
I can't install the magento/pwa-commerce package, so this list will have to be provided :)

Reply:
@nrdevau 

The only other module for commerce is `Magento_ContactGraphQlPwa`.